### PR TITLE
Split motor_and_servo.* into motors.* and servos.*

### PR DESCRIPTION
**Pull request by martinbudden**

### CODE REVIEW

File: src/main/config/parameter_group_ids.h [hunk 1]
```diff
@@ -66,6 +66,8 @@
 #define PG_MSP_SERVER_CONFIG 48
 #define PG_VOLTAGE_METER_CONFIG 49
 #define PG_AMPERAGE_METER_CONFIG 50
+#define PG_MOTOR_CONFIG 51
```
Comment (ledvinap) [inline src/main/config/parameter_group_ids.h]: Did you consider renaming PG_MOTOR_AND_SERVO_CONFIG to PG_MOTOR_CONFIG  and bumping record version?
I'm not sure which one is better ...

Adding dummy value (to replace servoCenterPulse) into motor config will even allow automatic upgrade ...

Comment (hydra) [inline src/main/config/parameter_group_ids.h]: @martinbudden yes, i think @ledvinap is probably right, more people use motors than servos so having automatic upgrade is probably a good idea.  call the dummy value 'reserved' or something similar.   

We would encounter an issue in the future if we were to use 'reserved' for something else without it being sanitized/reset.

Perhaps place a comment to remind us that if the value is used migrations should ignore any value stored in the 'reserved' field.

**Comment by hydra** [comment]
@martinbudden there is also a unit test failure.  See the travis build log.

**Comment by martinbudden** [comment]
Updated as per comments.

File: src/main/io/motors.h [hunk 2]
```diff
@@ -24,6 +24,7 @@ typedef struct motorConfig_s {
     uint16_t maxthrottle;                   // This is the maximum value for the ESCs at full power this value can be increased up to 2000
     uint16_t mincommand;                    // This is the value for the ESCs when they are not armed. In some cases, this value must be lowered down to 900 for some specific ESCs
     uint16_t motor_pwm_rate;                // The update rate of motor outputs (50-498Hz)
```
Comment (ledvinap) [inline src/main/io/motors.h]: `uint16_r reserved1;` between `mincommand` and `motor_pwm_rate` to take place of former `servoCenterPulse`. 
That way all motor fields stay valid after this change and there is no need to bump record version (and values are not reset)

**Comment by hydra** [comment]
merged

**Pull request closed without merging**